trunk/examples/witness-checking/regression/correctness/cohendiv-ll.c-witness.yaml

```diff
@@ -0,0 +1,54 @@
+-
```

Comment:
But I agree that it isn't nice. Unfortunately this is an issue with eo-yaml: https://github.com/decorators-squad/eo-yaml/issues/545
In 5eb799a I customized the output (somewhat hacky, since I need to work on strings) to fix this issue in our implementation.